src/styles/blocks/section1.scss

```diff
@@ -0,0 +1,37 @@
+.section_1 {
+  margin: 0;
+  padding: 32px 16px;
+  height: 134px;
+  width: 166px;
+  text-align: center;
+}
+
+.section_1_img {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 36px;
+}
+
+.section_1__text {
+  height: 36px;
+  width: 166px;
+  font-size: 12px;
+  font-weight: 500;
+  top: 175px;
```

Comment:
The 'top' property is used with positioned elements (like position: absolute, position: relative, etc.), but there is no position property specified here. If you want to position this element, you should add a position property with a value other than 'static'.